### DISCUSSION TITLE
Border

### DISCUSSION THREAD

**Discussion by yoonghm** [quarto-dev/quarto-cli#6852] [Q&A] [answered]
### Description

I have following format:

````qmd
```SQL
SELECT * FROM weather;
```

```TEXT
┌───────────────┬─────────┬─────────┬───────┬────────────┐
│     city      │ temp_lo │ temp_hi │ prcp  │    date    │
│    varchar    │  int32  │  int32  │ float │    date    │
├───────────────┼─────────┼─────────┼───────┼────────────┤
│ San Francisco │      46 │      50 │  0.25 │ 1994-11-27 │
│ San Francisco │      41 │      55 │   0.0 │ 1994-11-29 │
└───────────────┴─────────┴─────────┴───────┴────────────┘
```
````

The output is

![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/c4ef84ad-1b86-49e4-81a0-d093fe84e011)

The borders were omitted from the output.
 PDF
I manually crafted the following text
````qmd
```TEXT
┌──123────456────┬789
```
````

The output is 
![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/36fb6b11-cecc-4c4d-b08b-43829028ae83)

**Comment by cscheid** [comment]
If the monospace font doesn't have those glyphs, then there's unfortunately nothing quarto can do about that. I suspect that's the case (and you can verify by adding `keep-tex: true` and inspecting the .tex file)

**Comment by yoonghm** [comment]
Here is the portion from the `.tex` file:

````qmd
\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{SELECT}\NormalTok{ city, temp\_lo, temp\_hi, prcp, }\DataTypeTok{date}
\KeywordTok{FROM}\NormalTok{ weather;}
\end{Highlighting}
\end{Shaded}

The output should be:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{city           temp\_lo  temp\_hi  prcp   date}
\NormalTok{{-}{-}{-}{-}{-}{-}{-}{-}{-}{-}{-}{-}{-}  {-}{-}{-}{-}{-}{-}{-}  {-}{-}{-}{-}{-}{-}{-}  {-}{-}{-}{-}{-}  {-}{-}{-}{-}{-}{-}{-}{-}{-}{-}}
\NormalTok{San Francisco  46       50       0.25   1994{-}11{-}27}
\NormalTok{San Francisco  43       57       0.0    1994{-}11{-}29}
\NormalTok{Hayward        37       54              1994{-}11{-}29}
\NormalTok{San Francisco  46       50       0.25   1994{-}11{-}27}
\NormalTok{San Francisco  43       57       0.0    1994{-}11{-}29}
\NormalTok{Hayward        37       54              1994{-}11{-}29}
\end{Highlighting}
\end{Shaded}
````

I am not familiar with Latex, does it require special font? Thanks.

**Reply by mcanouil** [reply, marked as answer]
The bars are not ASCII, they are special glyphs, you need a font that has them.
For example, in my case, I can use Courier New
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/8429fcde-c129-407d-846a-9a2a2dbb5ee2">

**Comment by yoonghm** [comment]
Thanks
